compiler/passes/convert-typed-uast.cpp

```diff
@@ -4050,7 +4068,8 @@ void TConverter::ActualConverter::convertActual(const FormalActual& fa) {
     INT_ASSERT(astActual);
 
     // Convert the actual and leave.
-    std::get<Expr*>(slot) = tc_->convertExpr(astActual, rv_);
+    auto actualExpr = tc_->convertExpr(astActual, rv_);
+    std::get<Expr*>(slot) = tc_->storeInTempIfNeeded(actualExpr, {});
```

Comment:
Do we always need to have the actuals be temps?

Reply:
The uAST here could be a ``Call*``, and normalized production AST requires a temporary.

Reply:
Makes sense to me, thanks! Maybe we could have `storeInTempIfNeeded` be more sensitive so it can avoid that if possible.